doc/source/development/contributing_codebase.rst

```diff
@@ -807,8 +804,57 @@ across the number of specified cores or ``auto`` to utilize all the available co
    # Utilizes all available cores
    pytest -n auto pandas
 
-This can significantly reduce the time it takes to locally run tests before
-submitting a pull request.
+If you'd like to speed things along further a more advanced use of this
+command would look like this
+
+.. code-block:: bash
+
+    pytest pandas -n 4 -m "not skip-slow and not skip-network and not skip-db and not single_cpu" -r sxX
+
+In addition to the multithreaded performance increase this improves test
+speed by skipping some tests using the ``-m`` mark flag:
+
+- skip-slow: any test taking long (think seconds rather than milliseconds)
```

Comment:
Might make sense to remove the `skip-` from these labels since in the pytest examples we're removing `skip-`